package.json

```diff
@@ -58,7 +58,8 @@
     "lodash": "^4.17.15",
     "pretty-error": "^2.1.1",
     "tapable": "^1.1.3",
-    "util.promisify": "1.0.0"
+    "util.promisify": "1.0.0",
+    "webpack-import-plugins": "1.0.0"
```

Comment:
Will this add a lot of sub dependencies?
Will this increase the installation time & size a lot?

Reply:
Only 3 light libraries which are commonly used and usually hoisted.

https://github.com/rubeniskov/webpack-import-plugins/blob/3f920363e8dfab661bda4191caa1f83a8d37cde3/package.json#L35

```json
"dependencies": {
    "debug": "^4.2.0",
    "find-up": "^5.0.0",
    "resolve": "^1.18.1"
  },
```

Common modules which are using these `deps`:
`debug` -> [dependants(36975)](https://www.npmjs.com/browse/depended/debug) -> `@babel/core` is one of them
`find-up` -> [dependants(1718)](https://www.npmjs.com/browse/depended/find-up) -> `react-dev-utils` is one of them
`resolve` -> [dependants(5832)](https://www.npmjs.com/browse/depended/resolve) -> `@babel/core` is one of them

Reply:
any suggest?